File: ios/Classes/RazorpayFlutterPlugin.m
#import "RazorpayFlutterPlugin.h"
#import <razorpay_turbo_standard/razorpay_turbo_standard-Swift.h>

@implementation RazorpayFlutterPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar {
  [SwiftRazorpayFlutterPlugin registerWithRegistrar:registrar];
}
@end

File: ios/razorpay_flutter/Sources/razorpay_flutter/RazorpayFlutterPlugin.swift
import Flutter

/// Swift Package Manager entry point.
///
/// Flutter's generated registrant calls `[RazorpayFlutterPlugin registerWithRegistrar:]`, the
/// `pluginClass` from pubspec.yaml. Under CocoaPods that class is the Objective-C shim in
/// `ios/Classes`, which forwards to `SwiftRazorpayFlutterPlugin`. SwiftPM targets cannot mix
/// Objective-C and Swift, so this package instead exposes the same name as a subclass of the
/// shared Swift implementation. This file exists only in the SwiftPM package; the shared sources
/// next to it are symlinks into `ios/Classes`, so CocoaPods is unaffected.
@objc(RazorpayFlutterPlugin)
public final class RazorpayFlutterPlugin: SwiftRazorpayFlutterPlugin {}
